src/lib/icon/icon.scss

```diff
@@ -10,6 +10,13 @@ $mat-icon-size: 24px !default;
   fill: currentColor;
   height: $mat-icon-size;
   width: $mat-icon-size;
+
+  &.mat-icon-inline {
+    font-size: inherit;
```

Comment:
Does IE11 support `inherit`? Any reason for not using `1em`?

Reply:
FYI: `inherit` [is  supported](https://developer.mozilla.org/en-US/docs/Web/CSS/inherit) since IE8.

Reply:
Ah, I was thinking of `initial`

Reply:
We could use `1em` instead of `inherit` if you would like, I don't have a super strong preference at this point.

Reply:
`inherit` should be fine, I mixed up the IE support in my head